README.md

```diff
@@ -32,46 +32,41 @@ make -j4
 And that should give you `rbkit-app/RBKitClient` executable, which you can
 go ahead and run.
 
-## Qt build issue on OSX Mavericks
+## OSX Yosmite  
+Currenty there are 2 ways to compile rbkit-client on mac, with Qt Creator and with cmake.
 
-Qt binary distribution compiles with `-stdlib=libstdc++` for std lib,
-where as Xcode 5 on Mavericks compiles with `-stdlib=libc++`. So when you
-are using libraries compiled outside of Qt in the app, the linker will not
-be able to resolve the symbols in the stdlib. Fix this by doing to following.
-Go to `Projects` tab in Qt and under `Build Steps`, expand the qmake step and
-look for `-spec` argument. You need to add a flag `QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.9`
-in the `qmake.conf` file of that spec.
+1. Using Qt Creator
+   Double click the `rbkit-client/RbkitClient.pro` file to open it in the qt creator. In the left menu-bar click on `Debug` button and select `rbkit-app` under the Run field.  
+   Hit the Run button, after compilation the GUI should pop-up.
 
-For example:
+2. Using cmake
+   Running a shadow build, i.e building from a separate directory is very
+simple. Just create a build directory, say `~/rbkit-build`. Lets say
+source code is cloned at `~/rbkit-client`
 
-My qmake command is
+   ```bash
+   $ mkdir -p ~/rbkit-build
+   $ cd ~/rbkit-build
+   ```
 
-```bash
-qmake /Users/emil/OpenSource/rbkit-client/RbkitClient.pro -r -spec macx-g++
-CONFIG+=debug CONFIG+=x86_64 CONFIG+=declarative_debug CONFIG+=qml_debug
-```
+   Export `CMAKE_PREFIX_PATH`.  
+   For example, my `CMAKE_PREFIX_PATH` is:  
 
-Since value of `-spec` is macx-g++, I'll need to add a line with
-`QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.9` in `~/Qt5.2.1/5.2.1/clang_64/mkspecs/macx-g++/qmake.conf`
+   ```bash
+   $ export CMAKE_PREFIX_PATH=/Users/ishan/Qt5.4.2/5.4/clang_64/
+   ```
 
-*Note* - Above special syntax is usually not required on Linux systems.
+   run cmake
 
-Running a shadow build, i.e building from a separate directory is very
-simple. Just create a build directory, say `~/rbkit-build`. Lets say
-source code is cloned at `~/rbkit-client`
+   ```bash
+   $ cmake -DZMQ_BUILD_TESTS=OFF ../rbkit-client
```

Comment:
we can always edit CMakeLists.txt of zeromq, and turn this off. I haven't explored how to do it from top level CMakeLists.txt file. @ishankhare07 can you explore this?

cmake has to be run from build directory, just like how we run `configure` script from build directory. i don't thing there is an easy way to provide a default build right now.